Simplenote/src/main/java/com/automattic/simplenote/Simplenote.java

```diff
@@ -212,6 +218,30 @@ public boolean isInBackground() {
         return mIsInBackground;
     }
 
+    public void login(String email, String spToken) {
+        // Manually authorize the user with Simperium
+        User user = mSimperium.getUser();
```

Comment:
interesting. I didn't realize we could do this right here. is there _always_ a user object in Simperium that we modify?

Reply:
We don't know the internals of Simperium very well, this logic was actually copied from the logic with WPCom account, as it's similar in taking an email and a token, and authenticating the user with them.
But I can confirm that Simperium has always a user object that we can modify: https://github.com/Simperium/simperium-android/blob/4de51b2684da7bc76dfd625d1295b4bc0488a803/Simperium/src/main/java/com/simperium/Simperium.java#L74